src/views/List.jsx

```diff
@@ -1,15 +1,48 @@
 import { ListItem } from '../components';
+import { useState } from 'react';
 
 export function List({ data }) {
+	// creates a state variable to track searchbar input
+	const [query, setQuery] = useState('');
+
+	// sets the state to the searchbar input as the user types
+	const handleSearch = (e) => {
+		setQuery(e.target.value);
+	};
+
+	// clears the searchbar and resets the filtered list to the whole list
+	const clearFilter = (e) => {
+		setQuery('');
+	};
+
 	return (
 		<>
 			<p>
 				Hello from the <code>/list</code> page!
 			</p>
+			<form>
+				<label htmlFor="searchbar">
+					Filter Items
+					<input
+						type="text"
```

Comment:
I assumed it's not necessary because it's not being submitted anywhere, we're only grabbing the value to set state. Would be curious to know if it's best practice for other reasons to always include it.

Reply:
It's best practices in other places (typically with a 'q'), but because it's setting state it's not strictly necessary, as long as the input fields are bound to the corresponding state values and event handlers using value and onChange props.